extra/onnx_ops.py

```diff
@@ -66,10 +66,10 @@ def _padding(X, pads=None, auto_pad="NOTSET", axes=None, constant_value=0.):
   constant_padder = Tensor(np.pad(np.zeros(X.shape), np_pads, constant_values=constant_value), dtype=X.dtype)
   return zero_padded + constant_padder
 
-def Pad(x: Tensor, pads: Tensor, constant_value: Tensor=None, axes: Tensor=None, mode="constant"):
+def Pad(x: Tensor, pads: Tensor | tuple, constant_value: Tensor=None, axes: Tensor=None, mode="constant", value: float=0.):
```

Comment:
Cool, I didn't know you could do this!

Reply:
[new feature in `3.10`](https://docs.python.org/3/whatsnew/3.10.html#pep-604-new-type-union-operator). unless we're ready to bump v from `3.8`->`3.10`, we should use `Union` instead.